ivtest/regress-sv.list

```diff
@@ -980,6 +980,7 @@ br_gh840a               CE,-g2012               ivltests
 br_gh840b               CE,-g2012               ivltests
 br_gh979                normal,-g2012           ivltests
 bitsel_real_idx		CE,-g2012		ivltests gold=bitsel_real_idx.gold
+partsel_outside		normal,-g2005-sv	ivltests
```

Comment:
New tests should be added to the new python-based test framework (see https://steveicarus.github.io/iverilog/developer/regression_tests.html).

Reply:
OK, done.